Formula/macos-trash.rb

```diff
@@ -0,0 +1,23 @@
+class MacosTrash < Formula
+  desc "Move files and folders to the trash"
+  homepage "https://github.com/sindresorhus/macos-trash"
+  url "https://github.com/sindresorhus/macos-trash/archive/1.1.0.tar.gz"
+  sha256 "31c09d385bb50b0f76818a1fe2c850cf56b9575c9fa27ea963cba38dfaba7d04"
+  head "https://github.com/sindresorhus/macos-trash.git"
+
+  depends_on :xcode => ["11.0", :build]
```

Comment:
Retrospectively looking, I think your `xcode 11.0` already implied this has to be built with Mojave and later, not quite sure why we sill need `:macos => :yosemite`.

Also, it looks like this is the only formula using `:macos => :yosemite` :)